packages/data-wizard/src/statistics/index.ts

```diff
@@ -260,3 +260,142 @@ export function coefficientOfVariance(array: number[]): number {
   const arrayMean = mean(array);
   return stdev / arrayMean;
 }
+
+/**
+ * Return the sum of the array by specific measure.
+ * @param array - The array to process
+ * @param measure - The selected measure
+ */
+export function sumBy(data: any[], measure: string) {
```

Comment:
The first parameter in code comment is `array`, in other earlier funcs is `array`, but here it's `data`. Plz replace `data` with `array`. Or, replace all the `array` with `data`. The following codes has the same problem.